workers/unity/Packages/io.improbable.gdk.mobile/Editor/MobileLauncherWindow.cs

```diff
@@ -76,84 +130,107 @@ public void OnGUI()
 
                 CommonUIElements.DrawHorizontalLine(8, DarkGrey);
 
-                using (new GUILayout.HorizontalScope())
-                {
-                    using (new EditorGUI.DisabledScope(simulatorNames.Length == 0))
-                    {
-                        simulatorNameIndex =
-                            EditorGUILayout.Popup("Simulator Model", simulatorNameIndex, simulatorNames);
-                    }
+                DisplayDevicesAndLaunchButton(isAndroid: false, isEmulator: true);
 
-                    var buttonIcon = new GUIContent(EditorGUIUtility.IconContent("Refresh"))
-                    {
-                        tooltip = "Refresh your simulator list."
-                    };
+                CommonUIElements.DrawHorizontalLine(8, DarkGrey);
 
-                    if (GUILayout.Button(buttonIcon, EditorStyles.miniButton, GUILayout.ExpandWidth(false)))
-                    {
-                        simulatorNames = iOSLaunchUtils.RetrieveAvailableiOSSimulators().Keys.ToArray();
-                        simulatorNameIndex = 0;
-                    }
+                DisplayDevicesAndLaunchButton(isAndroid: false, isEmulator: false);
+            }
+        }
+#endif
+
+        private void DisplayDevicesAndLaunchButton(bool isAndroid, bool isEmulator)
```

Comment:
Can collapse these bools into an enum: 

```csharp
public enum DeviceType
{
	AndroidDevice = 0,
	AndroidEmulator = 1,
	iOSDevice = 2,
	iOSEmulator = 3
}
```

Then the below are switch-case statements rather than nested ternaries.